examples/README.md

```diff
@@ -0,0 +1,40 @@
+# Examples
+
+![types.svg](./types.svg)
+
+Gantry supports `services` (oval shapes) and `steps` (box shapes).
```

Comment:
I think this would be a great place to add a sentence or two about the difference between them

Reply:
I've added some explanation to this.

Reply:
Moved to the main readme as suggested.